docs/api/event-definitions-map.md

```diff
@@ -10,13 +10,10 @@ anotherEventDefinition,
 
 const eventDefinitionsMap = {
   'SOME_ACTION_TYPE': someEventDefinition,
-  'SOME_OTHER_ACTION_TYPE': [someOtherEventDefinition, anotherEventDefinition],
```

Comment:
Oh so instead of defining the multiple events in the definition map, you are deffering it to the function call. 

mhmm....

Aesthetically the new way is more pleasing. But less pure, since for every new unique tracking event, you'll have to build a new function.. or find a way to merge them together right? Probably have to use a third party library to build them together.